### PR TITLE
task 1 completed

### PR DESCRIPTION
**Pull request by Anurag-NURA**
Checking pull request

### CODE REVIEW

**Comment by GargPermil** [comment]
wrong base branch and not follow git workflow.

File: client/eslint.config.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    languageOptions: {
+      ecmaVersion: 2020,
+      globals: globals.browser,
+      parserOptions: {
+        ecmaVersion: 'latest',
+        ecmaFeatures: { jsx: true },
+        sourceType: 'module',
+      },
+    },
+    settings: { react: { version: '18.3' } },
```
Comment (GargPermil) [inline client/eslint.config.js:20]: Why using version 18.3

File: client/eslint.config.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    plugins: {
+      react,
+      'react-hooks': reactHooks,
+      'react-refresh': reactRefresh,
+    },
+    rules: {
+      ...js.configs.recommended.rules,
+      ...react.configs.recommended.rules,
+      ...react.configs['jsx-runtime'].rules,
+      ...reactHooks.configs.recommended.rules,
```
Comment (GargPermil) [inline client/eslint.config.js:30]: fgffdgdsg

File: client/package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  "name": "client",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint .",
+    "preview": "vite preview",
+    "prepare": "husky"
```
Comment (GargPermil) [inline client/package.json:11]: ewerweqr

File: client/src/App.css [hunk 4]
```diff
@@ -0,0 +1,7 @@
+.App {
+  position: relative;
+  width: 100vw;
```
Comment (GargPermil) [inline client/src/App.css:3]: wrqwerwqer

**Comment by Anurag-NURA** [comment]
I can't understand the problem

File: client/eslint.config.js (hunk 2 above)
Comment (Anurag-NURA) [reply client/eslint.config.js:30]: Looks good to me

**Pull request closed without merging**